src/steamship/app/request.py

```diff
@@ -36,7 +36,7 @@ class InvocationContext(CamelModel):
     invocable_type: str = None
 
 
-class Request(CamelModel):
+class Request(CamelModel):  # TODO (enias): Rename to ExecutableRequest?
```

Comment:
InvocableRequest? (given that that's the name you mentioned above in the PR)

Reply:
Yes agree! I'll feel it out today in another PR.